src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,235 @@
+                + "888    888 888               888    888                  888  .88P           888    \n"
+                + "888        88888b.   8888b.  888888 888888 888  888      8888888K.   .d88b.  888888 \n"
+                + "888        888 \"88b     \"88b 888    888    888  888      888  \"Y88b d88\"\"88b 888    "
+                + "\n" + "888    888 888  888 .d888888 888    888    888  888      888    888 888  888 888  "
+                + "  \n" + "Y88b  d88P 888  888 888  888 Y88b.  Y88b.  Y88b 888      "
+                + "888   d88P Y88..88P Y88b. " + " \n" + " \"Y8888P\"  888  888 \"Y888888  \"Y888  \"Y888  "
+                + "\"Y88888      8888888P\"   " + "\"Y88P\"   " + "\"Y888 \n" + "                   "
+                + "                             888" + "                            " + "     " + "\n" + "  "
+                + "                                       " + "  Y8b d88P                                 \n"
+                + "                                       "
+                + "     \"Y88P\"                                  ";
+
+        this.lines = "______________________________________________________";
+        this.defaultGreeting = this.lines + "\n" + "     Hello! I'm Chatty Bot ^~^ \n"
+                + "     What can I do for you?\n" + "Type 'help' if you wish to know more about"
+                + " what I can do!" + "\n" + lines + "\n";
+        this.commandParser = new Parser();
+    }
+
+    /**
+     * Gets and returns user inputs to be parsed.
+     *
+     * @return String to be parse
+     */
+    public String getInput() {
+        String nextLine = sc.nextLine();
+        return nextLine;
+    }
+
+    /**
+     * Returns welcome message to user.
+     *
+     * @return Welcome message.
+     */
+    public String printWelcome() {
+        return this.defaultGreeting;
+    }
+
+    /**
+     * Returns UI message when loading storage files.
+     *
+     * @return Load files message.
+     */
+    public String loadFile() {
+        String loadFileMessage = this.lines + "\n" + "     Previously saved list (if any) loaded."
+                + " You may enter your commands now:" + "\n" + this.lines;
+        return loadFileMessage;
+    }
+
+    /**
+     * Returns UI message when adding a new directory to store bot data.
+     *
+     * @return Directory added message.
+     */
+    public static String addDirectory() {
+        String addDirectorymessage = "______________________________________________________"
+                + "\n" + "     I see that you do not have a directory to store data. "
+                + "Created one for you before we proceed." + "\n"
+                + "______________________________________________________";
+        return addDirectorymessage;
+    }
+
+    /**
+     * Returns UI message when adding a new data file to store bot data.
+     *
+     * @return Adds data file message.
+     */
+    public static String addDataFile() {
```

Comment:
I like how you handle all Ui messages for user in Ui class 👍

Reply:
Thank you for the compliment :)